yalelibrary/yul-dc-fixture-manifests
# Adding viewingHint to Manifests of paged objects

**Ticket**

https://app.zenhub.com/workspaces/yul-dc-5e50083561915b11fc9e5850/issues/yalelibrary/yul-dc/115

**Change**

Adds a `viewingHint` of `paged` to the appropriate Manifests.  Updates manifests IDs and image references to `localhost` ports so that manifests should work with `yalelibrary/yul-dc-blacklight`

Note however that `yalelibrary/yul-dc-blacklight` is currently set up to use `yalelibraryit/dc-iiif-manifest:demo` (not `yalelibraryit/dc-iiif-manifest:latest`) so it will continue to pull the `demo` branch of this repo, which points to ECS.  

**Testing**
* check out the `viewing_hint` branch of the `yul-dc-manifest` project,  `docker-compose build` N.B. this tags image with `demo`
* run the `yul-dc-blacklight` project locally
* visit http://universalviewer.io/uv.html?manifest=http://127.0.0.1/manifests/10958722.json
